src/multiparser.js

```diff
@@ -12,6 +12,10 @@ const concat = docBuilders.concat;
 function printSubtree(subtreeParser, path, print, options) {
   const next = Object.assign({}, { transformDoc: doc => doc }, subtreeParser);
   next.options = Object.assign({}, options, next.options, {
+    trailingComma:
```

Comment:
I think this is clearer:
```js
if (next.options.parser === "json") {
  next.options.trailingComma = "none";
}
```
but up to you

Reply:
I usually prefer not mutating objects but it should be safe here. Thanks for the suggestion.